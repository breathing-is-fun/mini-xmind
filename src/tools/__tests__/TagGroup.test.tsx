import React from 'react';
import { mount } from 'enzyme';
let TagGroup: any;
switch (process.env.LIB_DIR) {
  case 'dist':
    TagGroup = require('../../../dist/lib/toolbar').default;
    break;
  case 'lib':
    TagGroup = require('../../../lib/tools').TagGroup;
    break;
  default:
    TagGroup = require('..').TagGroup;
    break;
}
import Listener from '../../utils/GlobalListener';
import 'nino-cli/scripts/setup';

describe('TagGroup', () => {
  beforeEach(() => {
    (window as any).DataCollector = new Listener();
  });

  afterEach(() => {
    (window as any).DataCollector = null;
  });

  it('when data is null, render correctly', () => {
    const wrapper = mount(<TagGroup />);
    expect(wrapper).toMatchSnapshot();
  });
});
