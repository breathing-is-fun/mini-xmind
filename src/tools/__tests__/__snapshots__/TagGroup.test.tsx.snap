// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagGroup render correctly 1`] = `<TagGroup />`;
